doc/nrf/releases_and_maturity/releases/release-notes-changelog.rst

```diff
@@ -38,7 +38,10 @@ IDE, OS, and tool support
 Board support
 =============
 
-|no_changes_yet_note|
+* Updated various tests and samples to use Zephyr's :ref:`native_sim<zephyr:native_sim>` instead of Zephyr's :ref:`native_posix<zephyr:native_posix>` for :ref:`running_unit_tests`.
```

Comment:
`Updated tests and samples`? Seems that you updated all of them

Reply:
imo, word `various` highlights that changes concern all areas of tests/samples. 
@greg-fer , what is your opinion?